From 883a0d2722e5b97da436d7bc11d17b4f69aa709e Mon Sep 17 00:00:00 2001
Message-ID: <883a0d2722e5b97da436d7bc11d17b4f69aa709e.1699318202.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1699318202.git.sharlatanus@gmail.com>
References: <cover.1699318202.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 4 Nov 2023 19:48:15 +0000
Subject: [PATCH 06/16] gnu: cl-metacopy: Update to 0.2.0-2.df7856f.

* gnu/packages/lisp-xyz.scm (cl-metacopy): Update to 0.2.0-2.df7856f.
[source]: Update URL as maintainer and owner of repository was changed.
[home-page]: As above.

Change-Id: I3c745dcc6355f98c9d11c908d010447dd8c6d295
---
 gnu/packages/lisp-xyz.scm | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 9fd4d509a4..24d84c5aff 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -21234,8 +21234,8 @@ (define-public cl-simple-inferiors
   (sbcl-package->cl-source-package sbcl-simple-inferiors))
 
 (define-public sbcl-metacopy
-  (let ((commit "1b5bf443206cc1dea7801ae23d1167bd02122d30")
-        (revision "1"))
+  (let ((commit "df7856f2a43fa91124fe780ef22f792040bc130c")
+        (revision "2"))
     (package
       (name "sbcl-metacopy")
       (version (git-version "0.2.0" revision commit))
@@ -21243,17 +21243,19 @@ (define-public sbcl-metacopy
        (origin
          (method git-fetch)
          (uri (git-reference
-               (url "https://github.com/gwkkwg/metacopy")
+               ;; Upstream changed the maintaner.
+               ;; legacy https://github.com/gwkkwg/metacopy
+               (url "https://github.com/hraban/metacopy")
                (commit commit)))
          (file-name (git-file-name "cl-metacopy" version))
          (sha256
-          (base32 "1rzp112djgw5n76s6hy2aq92bc43p0zd5bgzvqwvgvws4pls42s9"))))
+          (base32 "0l5ryg8pvqz9sn9s8bsvd0plvcnm7crsx86iqk5wxblkialaizip"))))
       (build-system asdf-build-system/sbcl)
       (native-inputs
        (list sbcl-asdf-system-connections sbcl-lift))
       (inputs
        (list sbcl-contextl sbcl-moptilities))
-      (home-page "https://github.com/gwkkwg/metacopy")
+      (home-page "https://github.com/hraban/metacopy")
       (synopsis "Flexible Common Lisp shallow/deep copy mechanism")
       (description
        "This package provides a flexible shallow/deep copy mechanism for
-- 
2.41.0

